.github/workflows/cloud-run-crons.yml

```diff
@@ -82,7 +82,7 @@ jobs:
+          --set-env-vars "CIO_DATABASE_URL=${{secrets.CIO_DATABASE_URL_EXTERNAL}},RFD_PDFS_IN_GITHUB=false,RFD_PDFS_IN_GOOGLE_DRIVE=false,RUST_BACKTRACE=1,RUST_LOG=info,GITHUB_ORG=oxidecomputer,GH_APP_ID=${{secrets.GH_APP_ID}},GH_PRIVATE_KEY=${{secrets.GH_PRIVATE_KEY}},SENDGRID_API_KEY=${{ secrets.SENDGRID_API_KEY }},SHIPPO_API_TOKEN=${{secrets.SHIPPO_API_TOKEN}},WEBHOOKY_SENTRY_DSN=${{secrets.WEBHOOKY_SENTRY_DSN}},GIT_HASH=${{ steps.extract_sha.outputs.hash }},SENTRY_ENV=production,DOCUSIGN_REDIRECT_URI=${{ secrets.DOCUSIGN_REDIRECT_URI }},DOCUSIGN_INTEGRATION_KEY=${{ secrets.DOCUSIGN_INTEGRATION_KEY }},DOCUSIGN_WEBHOOK_ENDPOINT=${{ secrets.DOCUSIGN_WEBHOOK_ENDPOINT }},DOCUSIGN_CLIENT_SECRET=${{ secrets.DOCUSIGN_CLIENT_SECRET }},GOOGLE_GEOCODE_API_KEY=${{ secrets.GOOGLE_GEOCODE_API_KEY}},RAMP_CLIENT_ID=${{ secrets.RAMP_CLIENT_ID }},RAMP_CLIENT_SECRET=${{secrets.RAMP_CLIENT_SECRET}},RAMP_REDIRECT_URI=${{ secrets.RAMP_REDIRECT_URI }},QUICKBOOKS_CLIENT_ID=${{ secrets.QUICKBOOKS_CLIENT_ID}},QUICKBOOKS_CLIENT_SECRET=${{secrets.QUICKBOOKS_CLIENT_SECRET}},QUICKBOOKS_REDIRECT_URI=${{secrets.QUICKBOOKS_REDIRECT_URI}},GUSTO_CLIENT_ID=${{secrets.GUSTO_CLIENT_ID}},GUSTO_CLIENT_SECRET=${{secrets.GUSTO_CLIENT_SECRET}},GUSTO_REDIRECT_URI=${{secrets.GUSTO_REDIRECT_URI}},GOOGLE_KEY_ENCODED=${{ secrets.GOOGLE_CIO_KEY_ENCODED }},MAILCHIMP_CLIENT_ID=${{ secrets.MAILCHIMP_CLIENT_ID }},MAILCHIMP_CLIENT_SECRET=${{ secrets.MAILCHIMP_CLIENT_SECRET }},MAILCHIMP_REDIRECT_URI=${{ secrets.MAILCHIMP_REDIRECT_URI }},SLACK_CLIENT_ID=${{ secrets.SLACK_CLIENT_ID }},SLACK_CLIENT_SECRET=${{secrets.SLACK_CLIENT_SECRET}},SLACK_REDIRECT_URI=${{secrets.SLACK_REDIRECT_URI}},ZOOM_CLIENT_ID=${{secrets.ZOOM_CLIENT_ID}},ZOOM_CLIENT_SECRET=${{secrets.ZOOM_CLIENT_SECRET}},ZOOM_REDIRECT_URI=${{secrets.ZOOM_REDIRECT_URI}},REVAI_API_KEY=${{secrets.REVAI_API_KEY}},MAILCHIMP_LIST_ID_RACK_LINE=${{secrets.MAILCHIMP_LIST_ID_RACK_LINE}},SHIPBOB_WEBHOOKS_URL=${{secrets.SHIPBOB_WEBHOOKS_URL}},EASYPOST_API_KEY=${{secrets.EASYPOST_API_KEY}},ZOHO_CLIENT_ID=${{secrets.ZOHO_CLIENT_ID}},ZOHO_CLIENT_SECRET=${{secrets.ZOHO_CLIENT_SECRET}},AIRTABLE_WH_KEY=${{secrets.AIRTABLE_WH_KEY}},DOCUSIGN_WH_KEY=${{secrets.DOCUSIGN_WH_KEY}},GH_WH_KEY=${{secrets.GH_WH_KEY}},INTERNAL_AUTH_BEARER=${{secrets.INTERNAL_AUTH_BEARER}},MAILCHIMP_WH_KEY=${{secrets.MAILCHIMP_WH_KEY}},SHIPPO_WH_KEY=${{secrets.SHIPPO_WH_KEY}},SLACK_WH_KEY=${{secrets.SLACK_WH_KEY}},MAILCHIMP_API_KEY=${{secrets.MAILCHIMP_API_KEY}},HIRING_AUTH_BEARER=${{secrets.HIRING_AUTH_BEARER}}" \
```

Comment:
Not something that needs to happen right now, but can we break these lines down? It's a bit hard to see what's happening. 

Alternatively we could use `--env-vars-file` and write all these values to a file before this step.

Reply:
Yeah, I can take a look at that. It is pretty impossible to parse by looking.